src/modules/navigation/components/PersonalInfoBlade/PersonalInfoBlade.tsx

```diff
@@ -22,9 +27,14 @@ const PersonalInfoBlade: FC<PersonalInfoBladeBladeProps> = ({
 	currentLayer,
 	refetch,
 }) => {
+	const user = useSelector(selectUser);
 	const { tText } = useTranslation();
 	const dispatch = useAppDispatch();
+	const { data: preferences } = useGetNewsletterPreferences(user?.email);
 
+	const [newsLetterChecked, setNewsLetterChecked] = useState<boolean>(
```

Comment:
keep consistant naming with existing functions
```suggestion
	const [newsletterChecked, setNewsletterChecked] = useState<boolean>(
```